docs/chaos/index.md

```diff
@@ -25,9 +25,9 @@ Using Polly helps introduce resilience to a project, but we don't want to have t
 * A way to revert easily, to control the blast radius.
 * To be production grade, to run this in a production or near-production system with automation.
 
-## Chaos strategies (a.k.a Monkey strategies)
+## Chaos strategies
 
-Chaos strategies (or Monkey strategies as we call them) are in essence a [Resilience strategy](../strategies/index.md#built-in-strategies), which means, as a *Resilience Strategy* is the minimum unit of resilience for Polly, a *Chaos Strategy* is the minimum unit of chaos for Simmy.
```

Comment:
Maybe leave a note that that's what they were referred to previously?

Reply:
Done